sql/rikai--0.1.sql

```diff
@@ -59,3 +59,10 @@ CREATE TRIGGER create_model
 AFTER INSERT ON ml.models
 FOR EACH ROW
 EXECUTE FUNCTION ml.create_model_trigger();
+
+-- User defined functions
+CREATE FUNCTION iou(box1 box, box2 box)
+RETURNS real
+PARALLEL SAFE
+AS $$ SELECT area(box1 # box2) / (area(box1) + area(box2) - area(box1 # box2)) $$
```

Comment:
`#` is bitwise xor? shouldn't it be `and`?

Reply:
`#` is for intersection , natively supported in postgres

https://www.postgresql.org/docs/14/functions-geometry.html